pystassh/session.py

```diff
@@ -51,6 +51,10 @@ def __init__(
         self._session = None
         self._channel = None
 
+    @property
```

Comment:
I could not find the use for this propery, so I removed it

Reply:
My bad, now that I'm writing the doc for this feature, I see the point!

Reply:
haha, yeup, it is the door to use channel's `read` and `write`